xml_converter/src/attribute/species_filter.cpp

```diff
@@ -1,9 +1,9 @@
 #include "species_filter.hpp"
 
 using namespace std;
-
-SpeciesFilter parse_RaceFilter(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors) {
-    SpeciesFilter filter;
-    filter.parse(input, errors);
-    return filter;
-}
+//  Functionality will change for parsing
```

Comment:
No nee to comment this out, just delete it. If we need access to what it used to be we can look back into the git history.